### ISSUE TITLE
How can I get slot2D to work?

### ISSUE THREAD

**Issue by gtdun** [CadQuery/cadquery#428]
_When I try something like:
import cadquery as cq
result = cq.Workplane("XY").slot2D(4, 1, 45).extrude(1)
show_object(result)
_I get the following error message:
AtributeError: 'Workplane' object has no atribute 'slot2D'

**Comment by jmwright** [comment]
You probably have an older version of CadQuery. It works fine for me on the latest master. The following is one way to install CadQuery to make sure you have all the latest features.
```
conda create -n cadquery-master
conda activate cadquery-master
conda install -c cadquery -c conda-forge cadquery=master

# If you want CQ-editor too
conda install -c cadquery -c conda-forge cq-editor=master
```

**Comment by jmwright** [comment]
@gtdun Is this working for you now?

**Comment by gtdun** [comment]
I have been able to use the slot2D with cadquery-2 not the cadquery-master.
I am unable to get a functioning conda "cadquery-master" install but I was able to with cadquery-2 from the cadquery installation instruction web site.

I also, am unable to get a functioning conda "cq-editor" install. It is unclear to me how to accomplish this. I am not sure if a separate env needs to be setup, or a specific python version is required, or if any other prerequisites are needed.

To answer your question I can now work with slot2D but I am unable to easily view or export my cq scripts without the cq-editor that will display the script.
